core/src/main/java/org/apache/calcite/avatica/MetaImpl.java

```diff
@@ -370,6 +371,27 @@ public MetaColumn(
         Integer charOctetLength,
         int ordinalPosition,
         String isNullable) {
+      this(tableCat, tableSchem, tableName, columnName, dataType, typeName, columnSize,
+          decimalDigits, numPrecRadix, nullable, charOctetLength, ordinalPosition, isNullable, "",
+          "");
+    }
+
+    public MetaColumn(
+        String tableCat,
+        String tableSchem,
+        String tableName,
+        String columnName,
+        int dataType,
+        String typeName,
+        Integer columnSize,
+        Integer decimalDigits,
+        Integer numPrecRadix,
+        int nullable,
+        Integer charOctetLength,
+        int ordinalPosition,
+        String isNullable,
+        String isAutoincrement,
```

Comment:
is auto-increment one word?  Do you camelCase hyphenated words? Interesting..

Reply:
https://stackoverflow.com/questions/44170873/camel-casing-hyphenated-english-words-e-g-re-render#:~:text=1%20Answer&text=In%20camel%2Dcase%2C%20individual%20trailing,a%20single%20word%20would%20normally. this SO user makes a compelling case, sounds good to me